pages/infrastructure_providers-validators/running_full_node.md

```diff
@@ -1,73 +1,59 @@
-# Running a Full Node
-
-### Save your Chain ID in `dydxprotocold` config
-
-Save the [chain-id](../infrastructure_providers-network/network_constants.mdx#chain-id). This will make it so you do not have to manually pass in the chain-id flag for every CLI command.
-
-```bash
-dydxprotocold config chain-id $CHAIN_ID
+# Run a Full Node
+Running a full node allows your system to participate in a dYdX chain network. The recommended settings and best practices on this page help ensure your node stays healthy and up to date with the network.
+
+> Code snippets on this page use example values. Replace them with your own. See the [Network Configuration](../infrastructure_providers-network/network_constants.mdx) section of the documentation for network constants and other resources you need to configure a full node.
+
+## Prerequisites
+You need a running, non-validating full node that is connected to network. 
+
+- If you created a system service for your node by following the instructions on the previous page, [Set Up a Full Node](../infrastructure_providers-validators/how_to_set_up_full_node.md), start your node with the following command:
+  ```bash
+  stystemctl start dydxprotocold
+  ```
+- To start your node with Cosmovisor, include the flag `--non-validating-full-node=true`. Cosmovisor may prompt you to configure additional variables in your environment or include them in your command.
+  > The flag `--non-validating-full-node` is required. It disables the functionality intended for validator nodes and enables additional logic for reading data.
+  ```bash
+  cosmovisor run start --non-validating-full-node=true 
+  ```
+- To start your node directly with the `dydxprotocold` binary, include the flag `--non-validating-full-node=true`. `dydxprotocold` may prompt you to configure additional variables in your environment or include them in your command.
+  > The flag `--non-validating-full-node` is required. It disables the functionality intended for validator nodes and enables additional logic for reading data.
```

Comment:
nit: this flag comes up very slightly earlier than this point, should we move up the disclaimer/note?

Reply:
Do you mean in the Set Up a Node page? That's a good call, I'm adding this note under the first time that flag comes up on the Set Up page

On this page I think this is the first time it comes up, although the note is below that instance. I think it makes sense to define it just *after* introducing the term so the user can see it in context first